src/main/drivers/rx_pwm.c

```diff
@@ -27,14 +27,14 @@
 
 #include "common/utils.h"
 
-#include "drivers/nvic.h"
 #include "drivers/io.h"
-#include "timer.h"
+#include "drivers/nvic.h"
+#include "drivers/pwm_output.h"
+#include "drivers/timer.h"
 
-#include "pwm_output.h"
-#include "rx_pwm.h"
+#include "pg/rx_pwm.h"
 
-#include "flight/mixer.h" //!!TODO remove dependency on this
+#include "rx_pwm.h"
```

Comment:
May as well change this to `drivers/rx_pwm.h`

Reply:
Until now I assumed it was Betaflight standard to have the `.h` file that corresponds to the `.c` file it is included in last, and without a prefix (so that the include stays correct if they get moved around).
I think this is a good convention. Are you sure we should be changing it?

Reply:
I hadn't noticed that convention. In that case, don't change it.

Reply:
Not sure it's a convention, since conventions in Betaflight are mostly tribal knowledge, but it seems to be done in a lot of files.

Reply:
Different question: What do you think of moving the `rx_` drivers into a subdirectory? (In general, I think subdirectorising more of `drivers/` would improve the repository structure.

Reply:
Yes, I'm in favour of splitting the drivers directory into more subdirectories.